source/backups.rst

```diff
@@ -176,6 +214,60 @@ Following steps are needed to restore a previously saved backup:
               backupName: backup1
             EOF
 
+.. _backups-pitr-restore:
+
+Restoring backup with point-in-time recovery
+********************************************
+
+Following steps are needed to to roll back the cluster to a
+specific date and time:
+
+Details:
+
+1. First of all make sure that the cluster is running.
+
+2. Now find out correct name for the **cluster** (backup name is not needed if
+   point-in-time recovery is enabled and will be ignored). Available
+   clusters can be listed with the following command:
+
+   .. code:: bash
+
+      kubectl get psmdb
+
+3. Edit the ``deploy/backup/restore.yaml`` file, adding the right cluster name
+   and additional restoration parameters to the ``pitr`` section
+   
+   .. code:: yaml
+   
+      spec:  
+        clusterName: my-cluster-name
+        pitr:
+          type: date
+          date: YYYY-MM-DD hh:mm:ss
+
+3. Run the actual restoration process:
+
+   .. code:: bash
+
+      kubectl apply -f deploy/backup/restore.yaml
+
+   .. note:: Storing backup settings in a separate file can be replaced by
+      passing its content to the ``kubectl apply`` command as follows:
+
+      .. code:: bash
+
+            cat <<EOF | kubectl apply -f-
+            apiVersion: psmdb.percona.com/v1
+            kind: PerconaServerMongoDBRestore
+            metadata:
+              name: restore1
+            spec:
+              clusterName: my-cluster-name
+              pitr:
+                type: date
+                date: YYYY-MM-DD hh:mm:ss
+            EOF
+
```

Comment:
I would love to mention here, that the user needs to take full backup for PITR to function. PBM will not opload if there is no full backup.
This is true for new clusters and also true for clusters which just recovered from backup.